release/readme.html

```diff
@@ -2,248 +2,424 @@
+            </div>
+            <div class="col-md-2"></div>
+        </div>
+    </div>
+</div>
+
+<div class="container">
+    <div class="row">
+        <div class="col-md-12" style="text-align:center;">
+            <a class="btn btn-primary margin-bottom-6" href="https://energyplus.net" role="button">
+                <span class="fa fa-external-link"></span> Go to the EnergyPlus Website
+            </a>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-12" style="text-align:center;">
+            <a class="btn btn-primary margin-bottom-6" href="https://energyplus.net/quickstart" role="button">
+                <span class="fa fa-external-link"></span> Go to the EnergyPlus Quick-Start Guide
+            </a>
+        </div>
+    </div>
+
+    <h2>Using EnergyPlus</h2>
+    <p>
+        Utilities and tools for creating, editing, and running input files that come with EnergyPlus include IDF Editor
+        and
+        EP-Launch. Many graphical user interfaces for EnergyPlus are available or under development. For the latest
+        tools,
+        please see
+        <a href="https://www.buildingenergysoftwaretools.com/?capabilities=Whole-building+Energy+Simulation&keywords=EnergyPlus">
+            this website for third party interfaces</a>. A completely new cross-platform version of EP-Launch has been
+        developed
+        and is available separate from the EnergyPlus install; see the <a
+            href="https://ep-launch.readthedocs.io/en/latest/intro.html#ep-launch">documentation</a> for more
+        information.
+        Some tools may produce out of date IDF files; you can then use the IDFVersionUpdater program to convert older
+        files to current EnergyPlus version (you may need to use the Multiple Version Conversions set – see below).
+    </p>
+
+    <h2><a id="Whats_going_on"></a>What’s going on in V9.2?</h2>
+    <div class="row">
+        <div class="col-md-12">
+            <a class="btn btn-primary margin-bottom-6" data-toggle="collapse" href="#cDifferent" role="button"
+               aria-expanded="false"
+               aria-controls="cDifferent">
+                Is 9.2 different from 9.1?
+            </a>
+            <div class="collapse" id="cDifferent">
+                <div class="card card-body">
+                    <h5 class="card-title">Yes... and no.</h5>
+                    <p class="card-text">The basic structure of EnergyPlus is still as it has been but there are some significant
+                    changes that you will want to be aware of. As always, there are a number of new features and bug
+                    fixes. The highlights of features for this version are listed below in this readme.
+                        Also, see the changelog for a complete list of new features added in this and previous versions.</p>
+                    <a href="https://github.com/NREL/EnergyPlus/releases/latest" class="card-link">ChangeLog</a>
```

Comment:
This changelog link points to the latest release which is fine until the next release happens. The one further below points to the local copy for this specific release. 
href="changelog.html">changelog</a>

Change this one to match?

Reply:
We could try, but unfortunately we have to change the link then before I actually make the final tag.  So it's a bit chicken/egg...